packages/amplify_analytics_pinpoint/example/.flutter-plugins

```diff
@@ -0,0 +1,8 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
Again - I don't this should be here. 

Same goes for all files in `amplify_analytics_pinpoint`.